### PR TITLE
feature(reserve-table): added reserve a table view controller

### PR DESCRIPTION
**Pull request by talbenasuli**
added reserve table view model

### CODE REVIEW

File: Dizzy/Screens/ReserveTable/ReserveTableVC.swift [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+            make.width.equalTo(tomorrowButton)
+            make.width.equalTo(otherButton)
+        }
+        
+        commentsTextView.snp.makeConstraints { make in
+            make.height.equalTo(Metrics.doublePadding * 3)
+        }
+        
+        closeButton.snp.makeConstraints { make in
+            make.top.equalToSuperview().inset(Metrics.doublePadding * 2)
```
Comment (menesh2) [inline Dizzy/Screens/ReserveTable/ReserveTableVC.swift]: dont use `*` here, you can set a constant or create new variable on `Metrics`

File: Dizzy/Screens/ReserveTable/ReserveTableVC.swift [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+            make.trailing.equalToSuperview().inset(Metrics.doublePadding)
+        }
+        
+        tonightButton.snp.makeConstraints { make in
+            make.width.equalTo(tomorrowButton)
+            make.width.equalTo(otherButton)
+        }
+        
+        commentsTextView.snp.makeConstraints { make in
+            make.height.equalTo(Metrics.doublePadding * 3)
```
Comment (menesh2) [inline Dizzy/Screens/ReserveTable/ReserveTableVC.swift]: dont use `*`

File: Dizzy/Screens/ReserveTable/ReserveTableVC.swift [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+        commentsTextView.textContainer.maximumNumberOfLines = 2
+        commentsTextView.textContainer.lineBreakMode = .byWordWrapping
+        commentsTextView.setBorder(borderColor: .lightGray)
+    }
+    
+    private func layoutViews() {
+        mainStackView.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(Metrics.padding)
+            make.leading.equalToSuperview().offset(Metrics.doublePadding)
+            make.trailing.equalToSuperview().inset(Metrics.doublePadding)
```
Comment (menesh2) [inline Dizzy/Screens/ReserveTable/ReserveTableVC.swift:167]: are you sure `inset` is not expanding the view? if do, dont use it

**Review by menesh2 — CHANGES_REQUESTED** [review]

File: Dizzy/Screens/ReserveTable/ReserveTableVC.swift (hunk 1 above)
Comment (talbenasuli) [reply Dizzy/Screens/ReserveTable/ReserveTableVC.swift]: Done

File: Dizzy/Screens/ReserveTable/ReserveTableVC.swift (hunk 2 above)
Comment (talbenasuli) [reply Dizzy/Screens/ReserveTable/ReserveTableVC.swift]: DOne

File: Dizzy/Screens/ReserveTable/ReserveTableVC.swift (hunk 3 above)
Comment (talbenasuli) [reply Dizzy/Screens/ReserveTable/ReserveTableVC.swift:167]: inset is qual to offset with minus number.

**Review by menesh2 — APPROVED** [review]

**Pull request merged**